src/store/actions/authActions.js

```diff
@@ -79,7 +82,8 @@ export const signOut = () => async (firebase, dispatch) => {
     await firebase.logout();
     window.location.reload();
   } catch (e) {
-    console.log(e.message);
+    console.log(e.
```

Comment:
are you using prettier?

Reply:
@Niweera I will push the commit after using prettier soon.

Reply:
@Niweera I have added commit with prettier. can you please review?